static/docs/user-guide/install/code.md

```diff
@@ -0,0 +1,51 @@
+# Install from code
```

Comment:
This is actually the Development section moved here and modified.
I am removing the last part that clones the code and installs from it, since this seems related to development, not related to usage.

I am also renaming it to "Install latest version".
A name that includes "development" would still give the wrong impression that this is for developers.